prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -209,23 +209,25 @@ public async Task<ActionResult> GetOrganizationClaimByOrgId(int organizationId)
         public async Task<ActionResult> ApproveOrganizationClaim(int organizationId, int claimId)
         {
             var orgClaim = await _organizationClaimService.GetOrganizationClaimAsync(claimId);
-            if (orgClaim == null || (orgClaim.OrganizationId != organizationId))
+            if (orgClaim == null || organizationId != orgClaim.OrganizationId)
             {
                 return NotFound("Cannot locate Claim for given Organization.");
             }
 
-            if (!await _organizationService.SwitchSigningAuthorityAsync(orgClaim.OrganizationId, orgClaim.NewSigningAuthorityId))
-            {
-                // TODO: Properly communicate error
-                return BadRequest("Could not assign Organization to new SigningAuthority.");
-            }
-
-            await _emailService.SendOrgClaimApprovalNotificationAsync(orgClaim);
-
-            await _businessEventService.CreateOrganizationEventAsync(orgClaim.OrganizationId, orgClaim.NewSigningAuthorityId, $"Organization Claim (Site ID/PEC provided: {orgClaim.ProvidedSiteId}, Reason: {orgClaim.Details}) approved.");
+            var existingSigningAuthorityId = await _organizationService.GetOrganizationSigningAuthorityIdAsync(organizationId);
+            var notificationRequired = existingSigningAuthorityId != orgClaim.NewSigningAuthorityId;
 
+            await _organizationService.SwitchSigningAuthorityAsync(organizationId, orgClaim.NewSigningAuthorityId);
+            await _organizationService.SwitchSitesProvisionerAsync(organizationId, orgClaim.NewSigningAuthorityId);
```

Comment:
@hogmann your PR will remove `SwitchSitesProvisionerAsync` call and method.  Temporary update for production deployment so after claiming endpoint calls aren't blocked by permissions issues with the provider.  Your update to look at Organization replaces it and then we can drop the provisioner column since we capture everything about site creation and claims in business events.